# Cease all operation, go to safe pose
bool all_hold

# Press throttle pedal?
bool drive_forward

# Change steering angle to x
float32 steering_wheel_angle

# Head tilt and pan
float32 head_pan
float32 head_tilt
